docs-src/go/chapter-durable-execution/why-add-the-replay-test.md

```diff
@@ -0,0 +1,25 @@
+---
+id: why-add-the-replay-test
+title: Why add a Replay test?
+description: The Replay test checks whether the current Workflow code (Workflow Definition) is compatible with earlier Workflow Execution's Event Histories.
+sidebar_label: why-replay-test
+tags:
+  - go sdk
+  - developer-guide-doc-type
+  - event history
+  - determinism
+---
+
+The Replay test is important because it verifies whether the current Workflow code (Workflow Definition) remains compatible with the Event Histories of earlier Workflow Executions.
```

Comment:
Just note, I debated _whether_ `whether` or `if` was the right word here.
I believe `whether` is correct. But commenting it here in case someone else has the same thought.
(Also, `or not` isn't needed either.)